### PR TITLE
168452151 allow multiple application insights to be created

### PR DESCRIPTION
**Pull request by giskarda**

### CODE REVIEW

File: westeurope/app_insights/terraform.tfvars [hunk 1]
```diff
@@ -15,7 +18,17 @@ terragrunt = {
...
-azurerm_application_insights_application_type  = "Web"
-azurerm_application_insights_name              = "01"
+applications_insights = [
+  {
+    azurerm_application_insights_application_type  = "web"
+    azurerm_application_insights_name              = "01"
+    description                                    = "To be determined"
+  },
+  {
+    azurerm_application_insights_application_type  = "other"
```
Comment (LucaPrete) [inline westeurope/app_insights/terraform.tfvars]: According to what we commented in modules, let's keep name, application_type (or type) and description

File: westeurope/app_insights/terraform.tfvars [hunk 2]
```diff
@@ -15,7 +18,17 @@ terragrunt = {
...
-# Notification Hub module variables
+# New infrastructure
 
-azurerm_application_insights_application_type  = "Web"
-azurerm_application_insights_name              = "01"
+applications_insights = [
+  {
+    azurerm_application_insights_application_type  = "web"
+    azurerm_application_insights_name              = "01"
+    description                                    = "To be determined"
```
Comment (LucaPrete) [inline westeurope/app_insights/terraform.tfvars]: Can we remove it for now if it still needs to be determined?

Comment (giskarda) [reply westeurope/app_insights/terraform.tfvars]: description is to help US figuring out what the application insight is. There is no description field unfortunately in the official resource.

I'm happy to remove it but that would mean we need to remember what is used for.

File: westeurope/app_insights/terraform.tfvars (hunk 1 above)
Comment (giskarda) [reply westeurope/app_insights/terraform.tfvars]: My understanding is that our policy is to keep a 1:1 relationship with original resource arguments and our wrappers. [1]

1. https://www.terraform.io/docs/providers/azurerm/r/application_insights.html

**Pull request merged**